redwood-layout-view/src/main/kotlin/app/cash/redwood/layout/view/ViewBox.kt

```diff
@@ -36,66 +34,91 @@ import app.cash.redwood.layout.widget.Box
 import app.cash.redwood.ui.Density
 import app.cash.redwood.ui.Margin
 import app.cash.redwood.widget.ViewGroupChildren
-import kotlin.math.roundToInt
 
 internal class ViewBox(
   context: Context,
-) : Box<View> {
+) : FrameLayout(context), Box<View> {
+  override var modifier: Modifier = Modifier
 
-  private val density = Density(context.resources)
+  override val value: Box<View> get() = this
```

Comment:
```suggestion
  override val value: View get() = this
```
Sorry I got it wrong first time. This is a `Widget<View>`'s `value` which would be typed `View`. 🤦‍♂️